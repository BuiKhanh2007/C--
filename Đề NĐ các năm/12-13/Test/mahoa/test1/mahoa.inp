4
fkiskru
gc cr  
gpdxd  
     fw
 x mk s
xl lpfi
uglimtg
