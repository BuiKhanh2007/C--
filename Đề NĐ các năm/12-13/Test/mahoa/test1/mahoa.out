jomwovy
kg gv  
kth|h  
     j{
 | qo w
|p ptjm
ykpmqxk
